questionnaires/phq8/phq8_armt_es.json

```diff
@@ -2,9 +2,9 @@
     {
         "field_name": "phq8_1",
         "form_name": "phq8",
-        "section_header": "¿con qué frecuencia, en las últimas dos semanas, ha tenido cualquiera de los siguientes problemas?",
+        "section_header": "¿Con qué frecuencia, en las últimas dos semanas, ha tenido cualquiera de los siguientes problemas?",
```

Comment:
>        "section_header": "¿Con qué frecuencia, en las últimas dos semanas, ha tenido cualquiera de los siguientes problemas?",
Should the `¿` remain I see it was removed for other lines -- but I've no idea?

Reply:
Yes, I think for the section header they want to keep the question mark but capitalize the `Con`, but for the options they want it removed, but I will check again with them. The changes requested are here: https://radar-base.atlassian.net/browse/RSD-144